src/Commands/ApartmentMakeCommand.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace BuildEmpire\Apartment\Commands;
+
+use BuildEmpire\Apartment\ArtisanApartmentCommands;
+use Illuminate\Console\Command;
+use BuildEmpire\Apartment\Exceptions\SchemaAlreadyExistsException;
+use Illuminate\Support\Composer;
+
+class ApartmentMakeCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'apartment:make {schemaName}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Add an apartment.';
+
+    /**
+     * Composer command.
+     *
+     * @var Composer
+     */
+    protected $composer;
+
+    /**
+     * ApartmentMakeCommand constructor.
+     * @param Composer $composer
+     */
+    public function __construct(Composer $composer)
+    {
+        parent::__construct();
+        $this->composer = $composer;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @param ArtisanApartmentCommands $artisanApartmentCommands
+     * @return bool
+     */
+    public function handle(ArtisanApartmentCommands $artisanApartmentCommands)
+    {
+        $schemaName = $this->argument('schemaName');
+
+        try {
+            $artisanApartmentCommands->makeSchema($schemaName);
+        } catch (SchemaAlreadyExistsException $e) {
```

Comment:
Yes, it's still gets logged within the lumen log file. But, the user gets a valid console error back. These are within the artisan command. So, it's better to get the feedback to the user within the console.